.github/workflows/ci.yaml

```diff
@@ -181,3 +181,52 @@ jobs:
                 env:
                     DATABASE_URL: "mysql://mysql:mysql@127.0.0.1:${{ job.services.mysql.ports[3306] }}/testdb"
                     DATABASE: "mysql://mysql:mysql@127.0.0.1:${{ job.services.mysql.ports[3306] }}/testdb"
+
+    solr-integration:
+        name: "Solr integration tests"
+        runs-on: "ubuntu-20.04"
+        timeout-minutes: 30
+        permissions:
+            packages: read
+            contents: read
+        strategy:
+            fail-fast: false
+            matrix:
+                php:
+                    - '7.4'
+                    - '8.0'
+                    - '8.1'
+        steps:
+            -   name: Start redis container
+                run: >- 
+                    docker run 
+                    --name redis 
+                    -p 6379:6379 
+                    -d 
+                    --entrypoint redis-server redis --appendfsync no --maxmemory 60mb --save ""
+    
+            - uses: actions/checkout@v2
+              with:
+                  fetch-depth: 0
+
+            - name: Setup PHP Action
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: 7.4
+                  coverage: none
+            
+            - name: Add solr dependency
+              run: composer require --no-update "ezsystems/ezplatform-solr-search-engine:dev-move-to-github-actions as 3.2.x-dev"
+
+            - uses: "ramsey/composer-install@v1"
+              with:
+                  dependency-versions: "highest"
+
+            - name: Init Solr
+              run: vendor/ezsystems/ezplatform-solr-search-engine/.github/init_solr.sh
```

Comment:
A pre-initialized solr container is already available:

```yaml
    solr-integration-tests:
        needs: tests
        runs-on: "ubuntu-20.04"
        timeout-minutes: 5
        permissions:
            packages: read
            contents: read

        services:
            solr:
                image: ghcr.io/ibexa/product-catalog/solr:latest
                ports:
                    - 8983
                options: >-
                    --health-cmd "solr status"
                    --health-interval 10s
                    --health-timeout 5s
                    --health-retries 10

```

Granted, build is not "proper" and the namespace of the container is incorrect, but it is usable / buildable.

Reply:
I've tried using it (see: https://github.com/ezsystems/ezplatform-kernel/runs/5373915671 , https://github.com/ezsystems/ezplatform-kernel/commit/39f4aa2448e6eeb137b23bfe94d193fb0b568e17#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR197-R209), but it didn't play nice:
```
Starting solr service container
  /usr/bin/docker --config /home/runner/work/_temp/.docker_858a99ea-2c09-4c40-956b-ec385f6ce7e7 login ghcr.io -u mnocon --password-stdin
  /usr/bin/docker --config /home/runner/work/_temp/.docker_858a99ea-2c09-4c40-956b-ec385f6ce7e7 pull ghcr.io/ibexa/product-catalog/solr:latest
  Error response from daemon: unauthorized
```

At that time I assumed it is not meant to be used in public repositories and dropped the idea, how to approach this?

Reply:
Reason this happens is that this image resides in a different repository. We would need to move to a public registry and/or into this repository.